worlds/banjo_tooie/__init__.py

```diff
@@ -1,6 +1,9 @@
 import logging
+from collections import Counter
 from math import ceil
 import time
+
+import loguru
```

Comment:
You should add a `requirements.txt` for this new external dependency instead of relying on it already being installed for Starcraft 2, although is it really needed here compared to the regular logger already used elsewhere?

Reply:
I think just `logging.warning` (which is already imported just above) should be fine for the use you had.